### ISSUE TITLE
Admissions Sales Consultant

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#2187]
Admissions Sales Consultant<br>
http://ift.tt/2q1ayB6<br><p><img alt="Logo.gif?ixlib=rails-2.1" src="http://ift.tt/2qaS94b"></p>
<p><strong>Headquarters:</strong> Chicago, IL<br><strong>URL:</strong> <a href="http://ift.tt/1X5Zr70">http://ift.tt/1X5Zr70</a></p>
<div><strong>Our Company</strong></div>
<div><br></div>
<div>Remote Year brings together communities of interesting professionals, entrepreneurs, and freelancers from across the globe to spend a year working, traveling, and exploring 12 cities around the world. Spending one month in each location, communities connect with local cultures and business ecosystems, forming lifelong, borderless personal and professional relationships along the way. Together we are shaping the future of work.</div>
<div><br></div>
<div><strong>Responsibilities</strong></div>
<div><br></div>
<ol>
<li>Hunt the legends who will join our communities by qualifying applicants, selecting the right ones, coaching them through the decision-making process to join us, and ultimately placing them in the community that best suits their needs.<br><br>
</li>
<li>You will strategize, execute, and be held accountable to weekly, monthly, and quarterly targets based on closing accepted applicants.<br><br>
</li>
<li>We are a data-driven organization. You will adapt to and utilize our current tracking systems, but will also have the confidence to use data to drive improvements in processes and priorities.<br><br>
</li>
<li>Zoom will be your new best friend. You will be spending a LOT of time interfacing and strategizing with potential Remotes and our team on video calls. You are at your best when you&rsquo;re waist-deep in work that fulfills you and you are energized by speaking with people all day long.<br><br>
</li>
<li>Communicate clearly and constantly with your team, and build crazy good relationships with them and potential Remotes via the magic of the interweb.<br><br>
</li>
<li>The role is balanced between operational excellence (sales and data analysis) and strategic work (evolving strategies and processes). You will be comfortable with both, and will also be willing and able to speak up and challenge the status quo in order to make things better.<br><br>
</li>
<li>Help accepted applicants build strategic proposals to present to their employers in support of their participation in Remote Year. Collaborate with these applicants on presentation strategy, manager discussions, and human resources conversations to facilitate program acceptance and organizational buy-in.<br><br>
</li>
<li>You must be a Remote Year expert, sharing the experience in a way that influences and impacts diverse applicants.<br><br>
</li>
</ol><div><strong>Must-have qualifications</strong></div>
<div><br></div>
<ul>
<li>Highly motivated and target-driven with 1-3 years proven track record in sales<br><br>
</li>
<li>Experience working with Salesforce or other CRM<br><br>
</li>
<li>Strong organizational skills, as well as a high level of attention to detail and efficiency while working under tight deadlines and on multiple initiatives simultaneously<br><br>
</li>
<li>Hunger to learn. Great isn't good enough. You must always be looking for ways to improve<br><br>
</li>
<li>Relationship management skills and openness to feedback<br><br>
</li>
<li>You must be comfortable with change. We are a startup and are constantly evolving<br><br>
</li>
<li>Experience traveling internationally<br><br>
</li>
<li>You must be able to travel full-time, contingent on your job performance and ability to meet and exceed your quota<br><br>
</li>
</ul><div><strong>Nice-to-have qualifications</strong></div>
<div><br></div>
<ul>
<li>Familiarity with Google office suite, Slack, and Zoom<br><br>
</li>
<li>Start-up experience<br><br>
</li>
<li>Experience working remotely<br><br>
</li>
<li>BS/BA degree or equivalent<br><br>
</li>
</ul><div><strong>Qualities</strong></div>
<div><br></div>
<ul>
<li>Passion for the cause &ndash; we need to get every potential Remote excited about the unique way that we are changing the world and you are the face of this<br><br>
</li>
<li>Passion for travel &ndash; role will require physical presence on a Remote Year program (contingent on performance)<br><br>
</li>
<li>Ownership &ndash; we don&rsquo;t like micromanaging; we expect dedication and ownership<br><br>
</li>
<li>Organization skills &ndash; great time management skills, ability to multitask<br><br>
</li>
<li>Interpersonal skills - this role is responsible for quickly conveying the Remote Year brand and experience to potential Remotes<br><br>
</li>
</ul><p><strong>To apply:</strong> Apply online at <a href="http://ift.tt/2pvX2bF">http://ift.tt/2pvX2bF</a></p>
<br><br>
via Remote Jobr Programming Jobs http://ift.tt/1WipnMk<br>
April 25, 2017 at 10:31AM
